core/arch/proc.h

```diff
@@ -102,12 +102,6 @@ enum {
  *   Intel IvyBridge           Family 6, Model 58 (0x3a)
```

Comment:
Please add some statement to the commit description about *why* this is being removed: old stale code for x86 emulation on Itanium which has not been touched in 10+ years and which is unlikely to ever be used again.

Reply:
Done.